js/north.js

```diff
@@ -186,6 +211,10 @@ jQuery( function ( $ ) {
 			$$.removeClass( 'to-close' );
 		});
 
+		if ( siteoriginNorth.smoothScroll ) {
+			$( '#mobile-navigation a[href*="#"]:not([href="#"])' ).smoothScroll();
```

Comment:
You'd also need to update the reference to the new `northSmoothScroll` function here, and on line 383.